pdc/apps/common/renderers.py

```diff
@@ -70,6 +70,9 @@
     - To sort by a field in descending order, prefix its name with minus (e.g. `-name`).
     - Use double underscores for nested field names (e.g. `parent__child` for `{"parent": {"child": ...}}`).
 """
+FIELDS_STRING = """
+ * `fields` (string) Comma separated list of fields to display (can improve performance).
+"""
```

Comment:
Remove " (can improve performance)" should be better, just we know this is enough.

Reply:
BTW: nice idea.

Reply:
I want to make it clear to user that it can really lead to much faster response from server. I understand that "can improve performance" is not very clear. Any ideas? Something like "can improve response time"?

Reply:
maybe, just "faster than geting all files"?